### PR TITLE
[16.0][MIG] commission_formula

### PR DESCRIPTION
**Pull request by andrel-exo**

### CODE REVIEW

**Comment by flalexg** [comment]
Hi @arleite , I think you left the module `sale_comission_pricelist` in the MR. Could you remove it? I saw the changes of this module here: https://github.com/OCA/commission/pull/425

**Comment by aliciagaarzo** [comment]
Hi @arleite ,
Reviewing the code I see it correct, except that I would launch the command to solve the pre-commit, kind regards.

**Review by aliciagaarzo — COMMENTED** [review]
Hi @arleite ,
Reviewing the code I see it correct, except that I would launch the command to solve the pre-commit, kind regards.

**Comment by aliciagaarzo** [comment]
Also, if possible, I would cover the tests to solve the codecov.

**Review by aliciagaarzo — APPROVED** [review]
Also, if possible, I would cover the tests to solve the codecov

File: commission_formula/views/commission_view.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        <field name="name">commission.form.inherit</field>
+        <field name="model">commission</field>
+        <field name="inherit_id" ref="commission.commission_form" />
+        <field name="arch" type="xml">
+
+            <xpath expr="//sheet/group[2]" position="after">
+                <group>
+                    <group>
+                        <field
+                            name="formula"
```
Comment (flalexg) [inline commission_formula/views/commission_view.xml:13]: Could we add the widget ace here? It would better I think.

**Review by flalexg — CHANGES_REQUESTED** [review]

**Review by flalexg — APPROVED** [review]
Functional and code review. LGTM!

**Pull request merged**